logdna/logdna.py

```diff
@@ -131,6 +143,7 @@ def flush(self):
                 self.flusher = threading.Timer(1, self.flush)
                 self.flusher.start()
 
+    # TODO(esatterwhite): complexity too high (14)
```

Comment:
This function I don't really want to touch right now, but it prevents linting from running. 
I think the maximum could be set to 10 (its 8 currently). That still leaves this.

I think maybe disable this rule until it can be fixt. I think having the linting in CI is better than not

Reply:
> I think having the linting in CI is better than not

100%